test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -1164,11 +1168,33 @@ def _revert_pbsconf_server(self, vals_to_set):
                 if restart_pbs:
                     # Restart all
                     server.pi.restart(server.hostname)
+                    if not server.isUp():
+                        self.fail("Server is not up")
+                    if not self.mom.isUp():
+                        self.fail("Mom is not up")
+                    if not self.scheduler.isUp():
+                        self.fail("Scheduler is not up")
+                    if not self.comm.isUp():
+                        self.fail("comm is not up")
+
                 else:
                     for initcmd in cmds_to_exec:
                         # start/stop the particular daemon
                         server.pi.initd(server.hostname, initcmd[1],
                                         daemon=initcmd[0])
+                        if initcmd[1] == "start":
+                            if initcmd[0] == "server":
+                                if not server.isUp():
+                                    self.fail("Server is not up")
+                            if initcmd[1] == "sched":
+                                if not self.sched.isUp():
```

Comment:
same comment here, you'll need to do server.mom.isUp(), server.scheduler.isUp() and server.comm.isUp()